src/Authentication/AuthenticationManager.php

```diff
@@ -224,7 +224,7 @@ public function handleHelloMessage(Realm $realm, Session $session, HelloMessage
         }
 
         // If no authentication providers are registered for this realm send an abort message
-        if ($this->realmHasAuthProvider($realm->getRealmName())) {
+        if ($this->realmHasAuthProvider($realm->getRealmName()) === false) {
```

Comment:
@Webonaute Has this actually caused an issue for you? `realmHasAuthProver` can only return a boolean:
https://github.com/voryx/Thruway/blob/7039e871bfedc319bcadf5445ac30e0fc0eb52be/src/Authentication/AuthenticationManager.php#L502-L513

Reply:
Read the comment just over the line than read the original if statement. :)

As soon as you set a auth it will never go in auth code under those line.

And somevar === false is lot more easy and clear to read than !somevar

Reply:
Link to my issue here https://github.com/voryx/ThruwayBundle/issues/88

Reply:
Link to my issue here https://github.com/voryx/ThruwayBundle/issues/88